<launch>
    <arg name="project" default="lvi_fusion"/>
    <!-- Load Parameters -->
    <!-- <rosparam file="$(find lvi_fusion)/config/param.yaml" command="load" /> -->

    <node pkg="$(arg project)" type="pub_data.py" name="pub_data" output="screen"/>
    <!-- Module SLAM -->
    <!-- <include file="$(find lvi_fusion)/launch/include/module_mapping.launch"/> -->
    <node pkg="$(arg project)" type="mapping2d" name="mapping2d" output="screen">
        <remap from="/scan" to="/scan"/>
        <remap from="/odometry/imu_incremental" to="/odom"/>
    </node>

    <!-- Robot publish TF -->
    <!-- <include file="$(find lvi_fusion)/launch/include/module_robot_state_publisher.launch"/> -->

    <!-- Save data to text file -->
    <!-- <include file="$(find lvi_fusion)/launch/include/module_saveData.launch"/> -->

    <!-- Show in Rviz -->
    <include file="$(find lvi_fusion)/launch/include/module_rviz.launch"/>

    <!-- Load data by rosbag -->
    <!-- <include file="$(find lvi_fusion)/launch/include/module_rosbag.launch"/> -->
    
    
</launch>